pkg/broker/logic.go

```diff
@@ -26,7 +26,7 @@ type BusinessLogic interface {
 	// For more information, see:
 	//
 	// https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#catalog-management
-	GetCatalog(w http.ResponseWriter, r *http.Request) (*osb.CatalogResponse, error)
+	GetCatalog(w http.ResponseWriter, r *http.Request, c *RequestContext) (*osb.CatalogResponse, error)
```

Comment:
I think we had a slight mismatch on this - I had in mind that `RequestContext` would contain the `ResponseWriter` and `http.Request`

Reply:
Sorry about that, I misunderstood. But I updated the PR for the `RequestContext` to contain the writer and request. Can you have a look again, thank you!